app/scripts/services/dimLoadoutService.factory.js

```diff
@@ -419,10 +419,10 @@
           });
         }
 
-        if (item) {
+        if (item && item.canBeEquippedBy(store)) {
```

Comment:
We should probably add non-equippables to a list so we can say "we didn't equip this" in a popup. We should also only check equippability if we're intending to equip it (rather than just move it).